proc/cutter.go

```diff
@@ -43,8 +43,11 @@ type Cutter struct {
 	complement  bool
 	cutBuilders map[int]*cutBuilder
 	fieldnames  []string
+	strict      bool
 }
 
+// NewCutter returns a Cutter whose Cut method returns a record if its input
+// record contains any field in fieldnames.
```

Comment:
This comment doesn't work for `complement=true`. How about something like

```suggestion
// NewCutter returns a Cutter whose Cut method returns a record if its input
// record contains any field in fieldname if complement is false. If complement 
// is true, the returned Cutter's Cut method returns a record for every input record.
```

Reply:
Fixed, I hope!